lib/kino_benchee.ex

```diff
@@ -1,2 +1,5 @@
 defmodule KinoBenchee do
+  @moduledoc """
+  DOCS GO HERE
```

Comment:
Could be useful to add a simple notebook to the docs, showing how an example benchmark is rendered. See [guides/components.livemd](https://github.com/livebook-dev/kino_vega_lite/blob/104da03dc68737f2ec4d5aaa2d226dce79a89957/guides/components.livemd) :)

Reply:
Added an example benchmark in the latest commit.